src/events/message_delete.rs

```diff
@@ -68,19 +68,50 @@ pub async fn message_delete(
         .map(|entry| entry.user_id)
         .and_then(|deletor| audit_logs.users.iter().find(|usr| usr.id == deletor));
 
+    let image = if let Some(_image) = msg
+        .attachments
+        .iter()
+        .filter(|a| a.height.is_some() && a.height.unwrap() > 0)
+        .next()
+    {
```

Comment:
you can use `.any` rather than `.filter(...).next().is_some()` (--> or the if let you did here). Also, I'm pretty sure that the height > 0 check should be unnecessary, as the attachment already only will have dimensions if it _is_ in fact an image (-> thus, `dimensions().is_some()` should work the same way)
```suggestion
    let image = if msg.attachments.iter().any(|a| a.dimensions().is_some()) {
```